server/database/seeds/02_create_reviews.ts

```
@@ -0,0 +1,72 @@
+import { each } from 'bluebird';
+import { randomUUID } from 'crypto';
+import * as Knex from 'knex';
+import _ from 'lodash';
+import { Model, ModelObject } from 'objection';
+
+import { appConfig } from '../../src/config';
+import { insertReview } from '../../src/functions';
+import { Course, Review, Semester, User } from '../../src/models';
+
+export async function seed(knex: Knex): Promise<void> {
+  Model.knex(knex);
+
+  if (appConfig.environment === 'production') {
+    return;
+  }
```

very important, thanks :) 

in fact, let's throw an Error here instead of silently dying